src/libraries/System.Collections.NonGeneric/README.md

```diff
@@ -0,0 +1,18 @@
+# System.Collections.NonGeneric
+This is the assembly that generally surfaces most non-generic collections such as [`Stack`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.stack) and [`Queue`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.queue).
+
+Non-generic collections that are used by lower-level parts of the framework, such as [`IList`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.ilist) and [`Hashtable`](https://learn.microsoft.com/en-us/dotnet/api/system.collections.hashtable) are surfaced by the `System.Runtime` assembly. The implementations for these collections live in [System.Private.Corelib](../System.Private.Corelib/src/System/Collections/).
+
+Documentation can be found at https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/concepts/collections.
+
+## Contribution Bar
+- [x] [We consider new features, new APIs and performance changes](../../libraries/README.md#primary-bar)
```

Comment:
Generally speaking, we don't consider new features for non-generic collections. They're largely in maintenance mode.